lib/alchemy/test_support/factories/attachment_factory.rb

```diff
@@ -2,7 +2,7 @@
 
 FactoryGirl.define do
   factory :alchemy_attachment, class: 'Alchemy::Attachment' do
-    file File.new(File.expand_path('../../../../../spec/fixtures/image.png', __FILE__))
+    image_file File.new(File.expand_path("#{Alchemy::Engine.root}/lib/alchemy/test_support/fixtures/image.png", __FILE__))
```

Comment:
I think, we should use `Alchemy::Engine.root.join()` instead of interpolating the string